releasenotes/notes/add-qfi-with-primitive-86d935d19dfff1a1.yaml

```diff
@@ -1,25 +1,31 @@
 ---
 features:
   - |
-    New Quantum Fisher Information (QFI) Algorithm using the primitives,
-    :class:`qiskit.algorithms.gradients.LinCombQFI`, has been added. It
-    internally uses the primitives to calculate the QFI.
+    New Quantum Geometric Tensor (QGT) and Quantum Fisher Information (QFI) Algorithm using the
+    primitives, :class:`qiskit.algorithms.gradients.LinCombQFI` and
+    :class:`qiskit.algorithms.gradients.LinCombQGT` have been added. They internally uses the
+    primitives to calculate.
 
     Example::
 
     .. code-block:: python
 
       from qiskit.circuit import QuantumCircuit, Parameter
-      from qiskit.algorithms.gradients import LinCombQFI
+      from qiskit.algorithms.gradients import LinCombQFI, LinCombQGT
 
       estimator = Estimator()
       a, b = Parameter("a"), Parameter("b")
       qc = QuantumCircuit(1)
       qc.h(0)
       qc.rz(a, 0)
       qc.rx(b, 0)
-      qfi = LinCombQFI(estimator)
+
       parameter_value = [[np.pi / 4, 0]]
+
+      qgt = LinCombQGT(estimator)
+      qgt_result = qgt.run([qc], parameter_value).result()
+
+      qfi = LinCombQFI(estimator)
```

Comment:
If we would delete the `LinCombQFI` class we also have to remember to remove this line here -- maybe we could instead show how to use the `QFI` object 🙂